src/SFA.DAS.Commitments.Database/StoredProcedures/CheckForOverlappingEmailsForTable.sql

```diff
@@ -14,19 +14,27 @@ BEGIN
 		A.EndDate,
 		A.IsApproved,
 		A.Email,
-		dbo.CourseDatesOverlap(A.StartDate, dbo.GetEndDateForOverlapChecks(A.PaymentStatus, A.EndDate, A.StopDate, A.CompletionDate), E.StartDate, E.EndDate) AS OverlapStatus
+		dbo.CourseDatesOverlap(
+            dbo.GetStartDateForOverlapChecks(A.IsOnFlexiPaymentPilot, A.StartDate, A.ActualStartDate),
+            dbo.GetEndDateForOverlapChecks(A.PaymentStatus, A.EndDate, A.StopDate, A.CompletionDate), 
+            E.StartDate, 
+            E.EndDate
+        ) AS OverlapStatus
 	FROM Apprenticeship A
 	JOIN @Emails E ON E.Email = A.Email
 	JOIN Commitment C ON A.CommitmentId = C.Id
 	WHERE 
 		CASE 
-			WHEN @CohortId IS NOT NULL AND A.CommitmentId = @CohortId AND A.IsApproved = 0 AND A.StartDate IS NOT NULL AND A.EndDate IS NOT NULL THEN 1
+			WHEN @CohortId IS NOT NULL AND A.CommitmentId = @CohortId AND A.IsApproved = 0 AND dbo.GetStartDateForOverlapChecks(A.IsOnFlexiPaymentPilot, A.StartDate, A.ActualStartDate) IS NOT NULL AND A.EndDate IS NOT NULL THEN 1
```

Comment:
@MikeV-TalentConsulting this is the line which I have changed. This when clause is for validating against draft apprenticeships and was previously using the StartDate property of a draft pilot apprenticeship which is null, resulting in ignoring all draft pilot apprenticeships when validating any other apprenticeship pilot or non pilot's email overlap.